dex/src/test/scala/com/wavesplatform/dex/actors/MatcherActorSpecification.scala

```diff
@@ -501,6 +504,7 @@ object MatcherActorSpecification {
     p.setAutoPilot { (sender: ActorRef, msg: Any) =>
       msg match {
         case _: Message.GetSnapshot => sender ! Response.GetSnapshot(None)
+        case _ => throw new IllegalArgumentException(s"Unexpected error")
```

Comment:
Let's write the `msg` in the error message + RuntimeException

Reply:
fixed